lib/ruby_indexer/test/constant_test.rb

```diff
@@ -27,8 +27,8 @@ class ::Bar
         end
       RUBY
 
-      assert_entry("FOO", Index::Entry::Constant, "/fake/path/foo.rb:0-0:0-8")
```

Comment:
Previous we were referencing the whole assignment, but I think it makes more sense to only reference the constant name?